doc/ref_cert/lfn/chapters/chapter03.md

```diff
@@ -17,28 +17,47 @@
   * [3.8.2 Resiliency Measurements](#3.8.2)
 * [3.9 NFVI Test Cases](#3.9)
 
+
 <a name="3.1"></a>
 ## 3.1 Introduction
 
-> Scope of this chapter is to have a list of test cases needed (a detailed table of sort)
+The objective of this chapter is to describe the requirements on NFVI test cases as derived from the reference model and architecture for the LFN-based compliance program. This set of requirements eventually determines the scope of the compliance program and the corresponding list of test cases included in the complaince program.  In particular, this chapter extends the generic list of NFVI test case requirements which is provided in Chapter 8.2.4 of the reference model.
+
 
 <a name="3.2"></a>
-## 3.2 Assumptions
+## 3.2 Generic Requirements
+
+All test cases must fullfil the generic requirements listed in Chapter 8.2.4 of the reference model.
+
+In addition, for test cases targeting the NFVI compliance program, the following requirements must be met:
+
+| Reference         | Description                                                                                                              |
+|-------------------|--------------------------------------------------------------------------------------------------------------------------|
+| x                 | All NFVI test cases *must* be automated, that is, implemented in an open source  test tool.                              |
+| x                 | All NFVI test cases *must* be integrated and run in the OPNFV CI/CD pipeline.                                            |
+| x                 | All NFVI test cases *must* exercise existing APIs only, that is, treat the NFVI platform as a black box.                 |
 
-Content to be written:
-- Assumptions:  Automatable, Integrated with CICD tool chain
 
 <a name="3.3"></a>
 ## 3.3 Requirement Type
 
 Content to be written:
 -  Type of requirement:  Bare metal, API, etc
 
+
 <a name="3.4"></a>
 ## 3.4 Profile Catalog
 
-Content to be written:
--  Table showing Profile Catalog
+Chapter 4.2 of the reference model defines three software profiles, targeting three different use cases:
+
+* Basic
+* Network intensive
+* Compute intensive
+
+The test cases selected for validating compliance of the three flavors must cover the functional and non-functional requirements as listed in Chapter 4.2.5 and 4.2.6 of the reference model.
```

Comment:
flavors -> profiles (in order to be consistent with the terminology defined other documents)

Reply:
Done. Updated. Resolved/Closed.